templates/kong.yml.erb

```diff
@@ -54,7 +54,7 @@ database: "<%= @database -%>"
 <% if @database == 'postgres' -%>
 postgres:
   host: "<%= @postgres_host -%>"
-  port: "<%= @postgres_port -%>"
+  port: <%= @postgres_port -%>
```

Comment:
This is eating the new line at the end due to the `-%>` ending. I think it should be just `<%= @postgres_port %>`.